extensions/elixir/src/language_servers/next_ls.rs

```diff
@@ -20,8 +20,12 @@ impl NextLs {
     pub fn language_server_binary_path(
         &mut self,
         language_server_id: &LanguageServerId,
-        _worktree: &zed::Worktree,
+        worktree: &zed::Worktree,
     ) -> Result<String> {
+        if let Some(path) = worktree.which("next-ls") {
```

Comment:
@andreicek this should be `nextls`

Reply:
Good point, I'll prepare a follow up right away.